src/app/pagamentoplanos/service/pagamentoplanos.service.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PagamentoplanosService {
+  public artPlusURL = `${environment.apiURL}`;
+  public token = localStorage.getItem('token');
+
+  httpOptions = {
+    headers: new HttpHeaders({
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${this.token}`,
+    }),
+  };
+
+  constructor(
+    private http: HttpClient,
+    private router: Router
+    ) { }
+
+  ngOnInit(): void {
+    if (this.token == undefined || this.token == null) {
+      this.router.navigate(['']);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8398929)
```suggestion
      this.router.navigate([""]);
```